content/advisories/2021-11-11-transaction-broadcast.zh-tw.md

```diff
@@ -0,0 +1,28 @@
+
++++
+title = "公告：對交易自動重新進行的回應"
+date = "2021-11-11"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+幣安今日暫停了狗狗幣的提款功能，並指出他們發現狗狗幣的＂小問題＂。對此我們想做出回應：
+
+數個月前 (請注意，在稍早發文中說的是一年前，但首次確認的提及是在四月) 幣安告知我們他們遇到一些交易卡住的情況，交易無法成功被礦工打包。我們建議幣安對這些交易使用 RBF (Replace-By-Fee) 機制 - 以使用較高手續費的新交易取代原先的交易。值得重視的是，RBF 能使先前的交易失效（因此也可說是＂取代＂），所以這個作法被優先建議。由於那些交易沒有啟用 RBF，我們建議手動建立使用相同 input 的新交易，強制使原先的交易失效。
+
+一段時間之後，幣安向我們表示他們遇到帳目核對問題。我們無法用幣安提供的資料重現那些狀況，但我們（在距今的幾個月前）建議了使用 `-zapwallettxes` 指令。值得注意的是，我們相信有使用此作法也就能防止該問題。
+
+昨日（11/10）我們被告知：在 Dogecoin Core 1.14.5 更新後，幣安先前卡住的交易都突然成功運行了。可能是因為 1.14.5 版本中交易手續費已被降低，致使之前有效但不被運行的交易被運行了。幣安提供給我們的唯一一筆交易案例，它的手續費自 1.14.5 起是有效的，但在 1.14.3 以前無效（因為太低）。請注意幣安在前幾天已從 1.14.3 直接升級為 1.14.5。
+
+目前所發生的這些情況，我們相信是升級版本後節點重啟時，之前卡住的交易自動被重新嘗試執行，並因為目前最低需求手續費已經降低而成功。這是交易手續費降低後正確的現象。
+
+## 學到的一課
+* 正確地取消交易：欲取消的交易和新的交易應使用相同 input 以使前者失效。
+  * 如果可以，使用 Replace-By-Fee 是更理想的作法。但手動建立一個使用相同 input 的新交易也能使先前的交易失效。
+* 請注意交易並沒有預定的逾時時限（timeout），但通常都會因記憶體限制而被終止。
+
+## 指引
+
+我們沒有再收到類似情況的回報。交易服務提供者若對停滯的無效交易有疑慮，我們建議停止節點，為以防萬一也請移除 mempool.dat 檔案，然後再以 `-zapwallettxes` 指令啟動節點。
```

Comment:
**The Foundation** has received no further reports of this happening. For any providers with concerns about stagnant invalid transactions, we recommend stopping the nodes, removing the mempool.dat file just in case, and then starting the node with **`-zapwallettxes`**.

-----

Please correct the highlighted sentences as above in the translation, according to the latest updates.